src/templates/settings.twig

```diff
@@ -0,0 +1,85 @@
+{#
+/**
+ * Website Toolbox Forum plugin for Craft CMS 3.x
+ * Website Toolbox Forum Settings.twig
+ * @author    Website Toolbox
+ * @copyright Copyright (c) 2020 Website Toolbox
+ * @link      https://websitetoolbox.com/
+ * @package   Website Toolbox Forum
+ * @since     3.0.0
+ */
+#}
```

Comment:
remove it.

Reply:
Its .twig file description. Its added in all .twig files